internal/controlplane/handlers_evalstatus.go

```diff
@@ -720,53 +699,3 @@ func dbSeverityToSeverity(dbSev db.Severity) (*minderv1.Severity, error) {
 
 	return severity, nil
 }
-
-// ugly function signature needed to handle the fact that we have two different
-// row types with the same fields, but no mechanism in the language to treat
-// them as the same thing.
-func getEntityName(
-	entityType db.Entities,
-	repoOwner sql.NullString,
-	repoName sql.NullString,
-	prNumber sql.NullInt64,
-	artifactName sql.NullString,
-) (string, error) {
-	switch entityType {
-	case db.EntitiesPullRequest:
-		if !repoOwner.Valid {
-			return "", errors.New("repo_owner is missing")
-		}
-		if !repoName.Valid {
-			return "", errors.New("repo_name is missing")
-		}
-		if !prNumber.Valid {
-			return "", errors.New("pr_number is missing")
-		}
-		return fmt.Sprintf("%s/%s#%d",
-			repoOwner.String,
-			repoName.String,
-			prNumber.Int64,
-		), nil
-	case db.EntitiesArtifact:
-		if !artifactName.Valid {
-			return "", errors.New("artifact_name is missing")
-		}
-		return artifactName.String, nil
```

Comment:
Note that we'll be using a differently formatted name for artifacts, will that be a problem? Otherwise it's fine to use the artifact's entity name here.

Reply:
I rather start using the new naming scheme already. Nothing is using artifacts just yet, right? We might want to double check that I don't break the smoke tests.